eng/pipelines/templates/steps/build-test.yml

```diff
@@ -50,32 +50,6 @@ steps:
     inputs:
       testResultsFiles: '**/*test*.xml'
       testRunTitle: '${{ parameters.OSName }} Python ${{ parameters.PythonVersion }}'
-
```

Comment:
I expanded above, the file still has the following variables being set:

```
RunMyPy: eq(variables['PythonVersion'], '3.7')
RunPylint: eq(variables['PythonVersion'], '3.7')
```
We should clean these up if they aren't being used now.

Reply:
Thanks. I have removed these unused variables.